File: app/src/main/java/social/entourage/android/tour/encounter/EncounterUploadCallback.java
package social.entourage.android.tour.encounter;

import social.entourage.android.api.model.map.Encounter;
import social.entourage.android.api.tape.EncounterTaskResult;

/**
 * Created by mihaiionescu on 04/05/2017.
 */

public interface EncounterUploadCallback {
    void onSuccess(Encounter encounter, EncounterTaskResult.OperationType operationType);

    void onFailure(Encounter encounter, EncounterTaskResult.OperationType operationType);
}

File: app/src/main/java/social/entourage/android/tour/encounter/EncounterUploadCallback.kt
package social.entourage.android.tour.encounter

import social.entourage.android.api.model.tour.Encounter
import social.entourage.android.api.tape.EncounterTaskResult.OperationType

/**
 * Created by mihaiionescu on 04/05/2017.
 */
interface EncounterUploadCallback {
    fun onSuccess(encounter: Encounter?, operationType: OperationType?)
    fun onFailure(encounter: Encounter?, operationType: OperationType?)
}